pkg/controller/clusterautoscaler/clusterautoscaler_test.go

```diff
@@ -169,9 +169,32 @@ func TestAutoscalerArgEnabled(t *testing.T) {
 	args := AutoscalerArgs(ca, &Config{CloudProvider: TestCloudProvider, Namespace: TestNamespace})
 
 	expected := []string{
-		fmt.Sprintf("--balance-similar-node-groups"),
-		fmt.Sprintf("--ignore-daemonsets-utilization"),
-		fmt.Sprintf("--skip-nodes-with-local-storage"),
+		fmt.Sprintf("--balance-similar-node-groups=true"),
+		fmt.Sprintf("--ignore-daemonsets-utilization=true"),
+		fmt.Sprintf("--skip-nodes-with-local-storage=true"),
```

Comment:
I know this is copying an existing pattern, but, why are we fmt printing strings with no format characters in them? Shouldn't we just use the strings raw?
```suggestion
		"--balance-similar-node-groups=true",
		"--ignore-daemonsets-utilization=true",
		"--skip-nodes-with-local-storage=true",
```

Reply:
i can change this up